app/assets/stylesheets/top_bar.scss

```diff
@@ -0,0 +1,67 @@
+@import "colors";
+$topbar-height: 4rem;
+
+nav.top-bar,
+nav.tab-bar,
+.contain-to-grid {
+  background: $dark-blue;
+  height: $topbar-height;
+
+  .top-bar-section li:not(.has-form) a:not(.button) {
+    background: $dark-blue;
+  }
+
+  .top-bar-section .dropdown li:not(.has-form):not(.active) {
+    & > a:not(.button) {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 5<br>Unnecessary parent selector (&)